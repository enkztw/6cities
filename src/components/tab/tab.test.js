import React from 'react';
import renderer from 'react-test-renderer';
import Tab from './tab.jsx';

const city = `Sochi`;


it(`<Tab /> should render "Sochi"`, () => {
  const tree = renderer.create(<Tab city={city} />).toJSON();

  expect(tree).toMatchSnapshot();
});
