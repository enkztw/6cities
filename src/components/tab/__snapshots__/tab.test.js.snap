// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tab /> should render "Sochi" 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item"
    href="#"
  >
    <span>
      Sochi
    </span>
  </a>
</li>
`;
